### DISCUSSION TITLE
How to report offensive repository names?

### DISCUSSION THREAD

**Discussion by domdfcoding** [community/community#65384] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Per the title. Under "Report this repository" I can only find an option for sexually offensive content, but that's not applicable to the repository I wish to report.

**Comment by talha4real** [comment]
The most direct way to report an offensive repository name is to contact GitHub Support. You can file a report through their [contact page](https://support.github.com/contact).
